### DISCUSSION TITLE
Installing HAOS with Armbian Bookworm 12 - Haade.fr

### DISCUSSION THREAD

**Discussion by giscus** [haade-administrator/haade.fr#46] [Comments]
# Installing HAOS with Armbian Bookworm 12 - Haade.fr

2024 - Tutorials installing home assistant OS on a cubietruck armv7 with Armbian version Debian 12 Bookworm all easily and without errors

https://haade.fr/en/blog/easy-installation-home-assistant-os-armbian-cubietruck-2024

**Comment by DenysZaytsev** [comment]
nice post
too bad I found it late, had to deal with all that manually serching for a solution in different sources. 
one thing to remember - armbian jammy doens't have systemd-resolved - it's not even int the repository, so I had to take tar from debian repo, prepare env and build it myself on my architecture (orange pi lite armhf).

**Reply by haade-administrator** [reply]
thank you a lot architecture a lot problem, [you can simply install with proxmox](https://haade.fr/en/blog/manufacture-a-high-performance-and-cheap-nas-amazon-renewed-and-proxmox-omv-truenas-home-assistant) into a virtual machine and test

**Comment by 532910** [comment]
it was hard to find, but NM requires netplan.io to be purged as it sets NM_UNMANAGED udev vav

**Comment by 532910** [comment]
PRETTY_NAME got rewritten with base-files update

**Comment by Muhammed83** [comment]
Hi Nico,

 I think this installation method is "Supervised Installation" , this is not like a Ha Os in raspberry which can be downloaded from Home Assistant website.

**Comment by haade-administrator** [comment]
This is a manual HAOS method like others and not a simple installation on Docker with only the supervisor
